### ISSUE TITLE
Is there a beginners guide to the GPTQ algorithm?

### ISSUE THREAD

**Issue by vgoklani** [IST-DASLab/gptq#32]
I found the paper hard to follow, is there a beginner / dummys guide? Perhaps slides from a talk or tutorial? Thanks!

**Comment by efrantar** [comment]
First, I would recommend to also take a look at our predecessor work [OBC](https://arxiv.org/abs/2208.11580) (which should explain some of the general ideas behind GPTQ) as well as our follow-up on sparsification [SparseGPT](https://arxiv.org/abs/2301.00774) (a related algorithm that can also be combined with GPTQ). Additionally, there is [this talk](https://www.youtube.com/watch?v=vhcb7hMyXwA) on YouTube by one of us which covers both GPTQ and SparseGPT.

There also seems to be [this](https://www.youtube.com/watch?v=05v2MA3CXKo) pretty detailed video on GPTQ specifically; though it is not associated with us.

I hope some of these pointers are helpful!
